eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -560,22 +561,42 @@ class EclairRpcClient(val instance: EclairInstance)(
 
   private var process: Option[Process] = None
 
-  def start(): Unit = {
+  def start(): Future[Unit] = {
 
-    if (process.isEmpty) {
-      val p = Process(
-        s"java -jar -Declair.datadir=${instance.authCredentials.datadir.get} $pathToEclairJar &")
-      val result = p.run()
-      logger.info(
-        s"Starting eclair with datadir ${instance.authCredentials.datadir.get}")
+    val _ = {
```

Comment:
Is there a reason this is wrapped in a block and not just on the top level?

Reply:
I think it logically groups all of these `Unit` together, as that is the return type of `_`